### ISSUE TITLE
Use sortlevel argument in evaluation

### ISSUE THREAD

**Issue by fschloesser** [GregorCH/ipet#28]
Currently 'sortlevel' it is set but not used.

**Comment by fschloesser** [comment]
What exactly is this option supposed to do?

**Comment by GregorCH** [comment]
Have you read the documentation of sortlevel?
```
sortlevel : level on which to base column sorting, '0' for group level, '1' for column level
```

**Comment by fschloesser** [comment]
I have, but i don't get it. And group level is not even used anymore.

**Comment by GregorCH** [comment]
The column headers consist of >= 1 levels, depending on attributes such as `indexsplit`.
The sortlevel functionality should allow to specify the level of the column keys, by which the resulting long table should be sorted. Implicitly, we currently use a sortlevel of 1, I guess.

**Comment by GregorCH** [comment]
You are right, group level is too specific and has been generalized in the mean time (because multiple levels of group keys are possible).

**Comment by fschloesser** [comment]
Thank you. So it should just be a pandas call to some kind of sortlevel func in a few lines, right?
Does this apply to both aggregated and long table?

**Comment by GregorCH** [comment]
Yes, there is a pandas function, (`sort_axis`, I guess). We are free to do whatever we want. I would suggest that it applies to both data frames, although the meaning would be a bit different for the aggregated table, because the column keys are used for the rows, as well.

**Comment by GregorCH** [comment]
There should be, IMHO, the possibilty not to sort, maybe by assigning `None` to the sortlevel (Note that this should be even the default behavior).

**Comment by fschloesser** [comment]
I have dealt with this in 0d5bd2ca19f03abf2bfcb59827398654ce158570

**Comment by GregorCH** [comment]
Why did you decide to push this directly instead of creating a merge request?

**Comment by fschloesser** [comment]
Because it was a simple thing (i thought). And in this project simple things seem to almost never be done via a pull request.
